### DISCUSSION TITLE
Breadcrumbs 🍞

### DISCUSSION THREAD

**Discussion by corinne-nagel** [twilio-labs/paste#1859] [Q&A] [answered]
Hi 👋  

I have breadcrumbs in my designs so that users can easily move from a [filtering view > room view > participant view](https://www.figma.com/proto/aUomFUTQZbQRuXMkIZylHl/Insights-2021-Q2?node-id=3227%3A78884&scaling=min-zoom&page-id=2070%3A40927&starting-point-node-id=3119%3A77023&show-proto-sidebar=1). This will allow them to easily navigate between levels of detail across the Insights "troubleshooting" product. There was a convo [here](https://twilio.slack.com/archives/C01DLUML16K/p1631297957206700) about what to do but while @johanna is out I'd like to get the conversation started about the pattern and added a comment in [the doc](https://docs.google.com/document/d/1hQ0S4qC0N8lwpVAo4ySv3woYHjTNMEyK2bPKKPZ1eBo/edit#)

Would love guidance on this ASAP as we're trying to build for Signal. Thanks!

**Comment by nkrantz** [comment, marked as answer]
Oh hey @corinne-nagel! Just want to confirm that you chatted with Katie/Sarah and that this will be handled by the Console side.

**Reply by kate-porter** [reply]
There's been a bit more convo in that Slack thread now, and this is definitely something that Console will need to tackle. They've created a spike ticket here: https://issues.corp.twilio.com/browse/CON-7079

**Reply by corinne-nagel** [reply]
Thanks!
